elftools/dwarf/compileunit.py

```diff
@@ -73,14 +76,84 @@ def get_top_DIE(self):
         """ Get the top DIE (which is either a DW_TAG_compile_unit or
             DW_TAG_partial_unit) of this CU
         """
-        return self._get_DIE(0)
+        dm = self._diemap
```

Comment:
Please don't do this, here and elsewhere. Just use `self._diemap` (and `self._dielist`) directly